src/js/media-time-display.js

```diff
@@ -12,8 +19,8 @@ const ButtonPressedKeys = ['Enter', ' '];
 const DEFAULT_TIMES_SEP = '&nbsp;/&nbsp;';
 
 const formatTimesLabel = (el, { timesSep = DEFAULT_TIMES_SEP } = {}) => {
-  const showRemaining = el.getAttribute('remaining') != null;
-  const showDuration = el.getAttribute('show-duration') != null;
+  const showRemaining = el.showRemaining;
```

Comment:
NOTE: This feels like scope creep for this round. Hoping for a gut check from others if you think I should drop it (still around in other branches).